### PR TITLE
Added: [Term Entry] Kotlin Math .atan2()

### LINKED ISSUE #3517: [Term Entry] Kotlin Math .atan2()
### Reason/inspiration (optional)

We would like a new term entry on the `.atan2()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/atan2/atan2.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by rahul0x00**
### Description

Added term entry for kotlin Math .atan2()
Closes #3517 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/atan2/atan2.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    val x = 3.0
+    val y = 4.0
+
+    val angle = Math.atan2(y, x)
+    println("The angle between the positive x-axis and the point ($x, $y) is $angle radians.")
+}
+```
+
+In this example, the .atan2() method calculates the angle, and the result will be printed as:
+`The angle between the positive x-axis and the point (3.0, 4.0) is 0.93 radians.`
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/atan2/atan2.md:42]:
````suggestion
In this example, the .atan2() method calculates the angle, and the result will output:

```shell
The angle between the positive x-axis and the point (3.0, 4.0) is 0.93 radians.
```
````
This is our preferred method of displaying example output.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@rahul0x00 I did a first review and only had one comment.

**Review by SSwiniarski — APPROVED** [review]
@rahul0x00 I make a couple of minor edits and this looks good for a second review.

File: content/kotlin/concepts/math-methods/terms/atan2/atan2.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- x: The x-coordinate of the point.
+- y: The y-coordinate of the point.
+
+The .atan2() method returns a value of type Double representing the angle in radians. The result is in the range [-π, π].
+
+## Example
+
+This example shows how use the .atan2() method to calculate the angle between the positive x-axis and a point with coordinates (3.0, 4.0):
+
+```
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/kotlin/concepts/math-methods/terms/atan2/atan2.md:32]: update code indentations to use 2 spaces not tabs

File: content/kotlin/concepts/math-methods/terms/atan2/atan2.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+fun main() {
+    val x = 3.0
+    val y = 4.0
+
+    val angle = Math.atan2(y, x)
+    println("The angle between the positive x-axis and the point ($x, $y) is $angle radians.")
+}
+```
+
+In this example, the .atan2() method calculates the angle, and the result will output:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atan2/atan2.md:42]:
```suggestion
The code above will result in the following output:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@rahul0x00 I've added my comments. Please update the entry, and tag me when you've completed the changes. Thanks

**Comment by rahul0x00** [comment]
@caupolicandiaz 
Hi, i've made the suggested changes

**Review by caupolicandiaz — APPROVED** [review]
@rahul0x00 @SSwiniarski LGTM

**Pull request merged**